tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -89,45 +113,50 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
 
       OsmNames.setOsmNames(feat, sf, 0);
 
-      if (sf.hasTag("place", "country")) {
+      if (place.equals("country")) {
         feat.setAttr("pmap:kind", "country")
+          // TODO: these should be from data join to Natural Earth, and if fail data join then default to 8
+          .setAttr("pmap:min_zoom", 1)
           .setZoomRange(0, 9);
-      } else if (sf.hasTag("place", "state", "province")) {
-        feat.setAttr("pmap:kind", "state")
+      } else if (place.equals("state") || place.equals("province")) {
+        feat.setAttr("pmap:kind", "region")
```

Comment:
Similarly Tilezen forward, `region` instead of `state`.

And `locality` instead of `city` below...